src/components/Home/DonationBanner.tsx

```diff
@@ -0,0 +1,33 @@
+import React, { useState } from "react";
+import DonationBannerContent from "./DonationBanner/DonationBannerContent";
+import DonationBannerStyle from "./DonationBanner.style";
+import { Col } from "antd";
+import { useTranslation } from "next-i18next";
+
+const DonationBanner = () => {
+  const { t } = useTranslation();
+  const [isBannerVisible, setIsBannerVisible] = useState(true);
+  const enableDonationBanner = process.env.NEXT_PUBLIC_ENABLE_BANNER_DONATION === "true";
```

Comment:
suggestion: this is looking great! now we need to create the deployment changes to include that variable in the containers